### ISSUE TITLE
Long press on ( table of contents) is not working. 

### ISSUE THREAD

**Issue by Zeyadas** [koreader/koreader#7414]
* KOReader version: latest 2.2021
* Device: Samsung galaxy Tab A 6 (android 5.1.1)

Long press on the (tablet of contents) does not reset or regenerate the tablet of contents, and the same issue is happening with the whole menu (it used to show a pop-up window describing the menu) now it just (flashes) on long press.

**Comment by poire-z** [comment]
Long-press on Table of contents has been removed as it was only known to people who read documentation/wiki/issues.
As [you said](https://github.com/koreader/koreader/issues/6938#issuecomment-737326995) that some People don't read docs (but it seems you're no longer following that great tradition :) we moved it into the Settings> submenu just below, with #7238, so it's a bit more visible.

No idea about the whole menu, some long-press should work. There might be not much, but the ones I put into `Gear>Tap and gestures>Links>` seem to still work.
